Makefile.tests

```diff
@@ -0,0 +1,43 @@
+##
+# Run all the tests.
+#
+#  make run-tests
+#
+# run a selected suite of tests
+#
+#  make run-tests SUITE=tests/test_cosmology
+#
+# To debug a test
+#
+#  make tests/test_cosmology
+#  gdb tests/test_cosmology
+#
+##
+TESTED = \
+	interp \
+	powerspectrum \
+	cosmology \
+	timefac \
+	timebinmgr
+
+TESTBIN = $(TESTED:%=tests/test_%)
+SUITE?= $(TESTBIN)
+
+.PHONY: run-tests build-tests
+
+
+depends/lib/libcmocka.a:
+	cd depends; make lib/libcmocka.a
+
+build-tests: $(TESTBIN)
+
+run-tests: build-tests
```

Comment:
Can you make this just "make test" as before? That is a standard make target name.

Reply:
We use `run-tests.py` in many python packages -- that's where the name is from. I guess make is already a verb then it doesn't make sense to have two verbs.